src/main/kotlin/org/jetbrains/exposed/dao/UUIDEntity.kt

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.exposed.dao
+
+import org.jetbrains.exposed.sql.Column
+import java.util.UUID
+
+open class UUIDTable(name: String = "", columName: String = "id") : IdTable<String>(name) {
+  override val id: Column<EntityID<String>> = varchar(columName, 36)
```

Comment:
Why use `String` when there is support for `UUID` OOTB? You can use `uuid(columnName)` instead, and adapt the rest accordingly.

I was going to do this myself, but saw your PR, so I guess it's better to fix this one instead, if you don't mind.